cilium-cli/connectivity/builder/egress_gateway_with_l7_policy.go

```diff
@@ -21,7 +21,7 @@ var clientEgressL7HTTPExternalYAML string
 type egressGatewayWithL7Policy struct{}
 
 func (t egressGatewayWithL7Policy) build(ct *check.ConnectivityTest, templates map[string]string) {
-	newTest("egress-gateway-with-l7-policy", ct).
```

Comment:
How come this is an issue? We don't run this test in the IPsec workflows given we don't enable egress gateway there today... :confused: